cors/backend/README.md

```diff
@@ -0,0 +1,26 @@
+# actix_cors::Cors
+
+## Running Server
+
+```sh
+cd cors/backend
+cargo run
+# starting HTTP server at http://localhost:8080
+```
+
+### web client
+
+- [http://localhost:8080/user/info](http://localhost:8080/user/info)
+  ```json
+  payload structure
+  {
+    "username": "username",
+    "email": "email",
+    "password": "password",
+    "confirm_password": "password"
+  }
+  ```
```

Comment:
```suggestion
  ```json5
  // payload structure
  {
    "username": "username",
    "email": "email",
    "password": "password",
    "confirm_password": "password"
  }
  ```
```

Reply:
Thank you for your suggestion. I will work on this 🤩